services/kafka.go

```diff
@@ -1,21 +1,33 @@
 package services
 
 import (
+	"encoding/json"
+	"log"
+	"os"
+	"time"
+
 	"github.com/Shopify/sarama"
+	"github.com/digicatapult/wasp-ingest-rtmp/util"
+)
+
+const (
+	KafkaTopicEnv = "KAFKA_TOPIC"
```

Comment:
what do you think about moving this to utils/env.go?

Reply:
yeah I was torn about this, better to have them in one place